rotkehlchen/chain/ethereum/typing.py

```diff
@@ -108,9 +108,31 @@ def __hash__(self) -> int:
     str,  # amount_deposited
 ]
 
+# When reading from the database we take into account the ownership proportion
+ValidatorDailyStatsDBTupleRead = Tuple[
+    int,  # validator index
+    int,  # timestamp
+    str,  # usd_price_start
+    str,  # usd_price_end
+    str,  # pnl_amount
+    str,  # start_balance
+    str,  # end_balance
+    int,  # missed_attestations
+    int,  # orphaned_attestations
+    int,  # proposed_blocks
+    int,  # missed_blocks
+    int,  # orphaned_blocks
+    int,  # included_attester_slashings
+    int,  # proposer_attester_slashings
+    int,  # deposits_number
+    str,  # amount_deposited
+    int,  # ownership_proportion
+]
+
 
 class ValidatorDailyStats(NamedTuple):
     validator_index: int  # keeping the index here so it can be shown in accounting
+    ownership_proportion: FVal  # Keeping this so it can be used during accounting
```

Comment:
This is not needed. We already know the ownership proportion since we have it in the validator. We are not able to get, or interested to even try guessing historic ownership of validators
```suggestion
```

But that also means my query of summing PnL for the frontend that I just added in the other PR will need to become more complicated :sweat: